scripts/build-and-publish-docker

```diff
@@ -39,7 +39,7 @@ for container in pulumi actions; do
     docker build --build-arg PULUMI_VERSION="${CLI_VERSION}" \
         -t "pulumi/${container}:${CLI_VERSION}" \
         -t "pulumi/${container}:latest" \
-        "${SCRIPT_DIR}/../dist/docker"
+        "${SCRIPT_DIR}/../dist/${container}"
```

Comment:
This will break the `pulumi/pulumi` image right - as it is in `../dist/docker` not `../dist/pulumi`?

Reply:
Good catch, I missed that.

Yes, that would break things 😬 . I'll rename the `dist/docker` folder to `dist/pulumi` so that they match the names of the container images, unless there is some reason not to.

Reply:
Minor - but should we put these in `dist/docker/actions/Dockerfile` and `dist/docker/pulumi/Dockerfile` to make it more clear these are the docker distributions?

Reply:
> should we put these in ... make it more clear these are the docker distributions?

That's a good idea, but I'm inclined to leave it as-is for now, since having the only folder in `dist` be `docker` seems like were just over-categorizing things. (... and also sidesteps the equally reasonable notion of just renaming `dist` to `docker`.)